src/NServiceBus.Core.Tests/API/APIApprovals.ApproveNServiceBus.approved.txt

```diff
@@ -606,6 +606,7 @@ namespace NServiceBus
         void Customize(NServiceBus.EndpointConfiguration configuration);
     }
     public class InMemoryPersistence : NServiceBus.Persistence.PersistenceDefinition { }
+    public interface INotification { }
```

Comment:
Does the marker interface provide any kind of value? In theory it could be anything